Connected System Plug-in (CSP) Examples/Hello World/src/main/java/com/mycorp/helloworld/templates/HelloWorldConnectedSystemTemplate.java

```diff
@@ -4,15 +4,18 @@
 import com.appian.connectedsystems.simplified.sdk.configuration.SimpleConfiguration;
 import com.appian.connectedsystems.templateframework.sdk.ExecutionContext;
 import com.appian.connectedsystems.templateframework.sdk.TemplateId;
+import java.util.*;
 
 @TemplateId(name="HelloWorldConnectedSystemTemplate")
 public class HelloWorldConnectedSystemTemplate extends SimpleConnectedSystemTemplate {
 
   public static final String CS_PROP_KEY = "csProp";
+  public String PASSWORD = "password";
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') 'PASSWORD' detected in this expression, review this potentially hard-coded credential. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS2068)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make PASSWORD a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "PASSWORD" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS00116)